dev-util/sysdig-kmod/files/0.27.1-fix-kmod-build-on-5.18+.patch

```diff
@@ -0,0 +1,59 @@
+From 36e945af4feb31c8b875f7d4624592b0c1ff929d Mon Sep 17 00:00:00 2001
```

Comment:
At the top of the patch, please include a link to the relevant Gentoo bug, and upstream link(s), including where you found the commit.

Reply:
Done!